pallets/free-proxy/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2021"
 license = "GPL-3.0-only"
 homepage = "https://subsocial.network"
 repository = "https://github.com/dappforce/subsocial-parachain"
+description = "Module for allowing accounts to get free proxy."
```

Comment:
Why is it module but not pallet?

Reply:
Most of our pallet descriptions follows this Module. So i kept it

Reply:
Sometimes we call them modules, and sometimes pallets. Not good for consistency.

Reply:
Changed to pallet, it should be clear now.